### DISCUSSION TITLE
0.0.55

### DISCUSSION THREAD

**Discussion by Xzandro** [Xzandro/sw-exporter#496] [General]
This release is mainly to recover Profile file exports. com2us added a new encryption for pretty much all events, besides the one that is necessary for the profile file. But from this release on, it will not contain the sealed monster storage, since that event is also affected by the new encryption.

- Disabled Merge sealed monster storage
- Added new rune set icons

<hr /><em>This discussion was created from the release <a href='https://github.com/Xzandro/sw-exporter/releases/tag/v0.0.55'>0.0.55</a>.</em>

**Comment by moatftw** [comment]
Anyone still have this error in this version?
Error decrypting request data - ignoring. Error: error:1e000065:Cipher functions:OPENSSL_internal:BAD_DECRYPT
I am still having this. I regen the cert already.
Can not dump the profile json file and I have a plugin written to save RTA replays which can't work either.

**Reply by omgitzdeadpool** [reply]
i still have the same error
![Screenshot 2023-09-14 182755](https://github.com/Xzandro/sw-exporter/assets/145072674/1d9e0fe4-c5f5-453e-a873-6702c53b1549)

**Comment by moatftw** [comment]
from the commit log I see I have to disable the option: "Merge sealed monster storage into profile data" to get the profile json dump working. 
Is there any hope to get the RTA replay info decrypted again?

**Comment by uncompleted** [comment]
All plugins are not working

**Reply by Xzandro** [reply]
https://www.reddit.com/r/summonerswar/comments/16idmio/state_of_swex_after_patch_809/

**Comment by ghost** [comment]
I can start my sw with the proxy activate but the data can't be store in json, someone have any idea ?

**Reply by loicgreffier** [reply]
Disabling "Merge sealed monster storage into profile data" in the Summoners War Exporter settings does the trick for now

**Reply by ghost** [reply]
ok the problem was with the https. With my iphone i can't start the game and get my json but with my emulator i can have the json

**Comment by MrSuricate2** [comment]
Will the application be compatible with the google play games beta installer on windows?

**Reply by Xzandro** [reply]
Probably not since it is impossible to set a proxy for Google Play games. Steam version in January could work tho, when they use the global Windows proxy. Thats still needs to be seen.

**Reply by omgitzdeadpool** [reply]
i am happy that it is working again, but question on the json file, as when i upload the file...it does not appear that is it correct as there are alot of ruins that i have upgraded or grinded and the info is not showing when i import my json to the SWARFARM site...any thoughts?

**Comment by G1996m** [comment]
I haven't downloaded json files for awhile.  I have V0.0.55 Exporter.  My certificate was out of date so I got a new certificate, but the current one that I can download says it expired 11/6/2023.  Where can I obtain one that hasn't expired?

**Reply by G1996m** [reply]
I figured out what I was doing wrong.  I was just clicking "Get a Cert", rather than going to the Settings and clicking "Regenerate Cert".
